addons/website/models/mixins.py

```diff
@@ -611,6 +611,19 @@ class WebsiteSearchableMixin(models.AbstractModel):
     _name = 'website.searchable.mixin'
     _description = 'Website Searchable Mixin'
 
+    @api.model
+    def _search_get_required_search_terms_rule(self, number_of_terms):
```

Comment:
I feel this is better fits here.
```suggestion
    def _search_get_required_search_terms(self, number_of_terms):
```
OR `_search_get_matching_threshold`